### ISSUE TITLE
Odd formatting of /collections/{collection_id} 404 response

### ISSUE THREAD

**Issue by duckontheweb** [stac-utils/stac-fastapi#419]
This is the response when trying to get a collection that does not exist (using the PgSTAC backend):

```console
$ curl http://api.dev.internal:9089/data-catalog-api/collections/optis_v2_1_0_WinterCoverCrop_2014
{"detail":"Collection <built-in function id> does not exist."}
```

It looks like the culprit is using `id` instead of `collection_id` [here](https://github.com/stac-utils/stac-fastapi/blob/226bdc0b20474c5d81d273abee4d10bc13874977/stac_fastapi/pgstac/stac_fastapi/pgstac/core.py#L101).

**Comment by alukach** [comment]
Just ran into this as well, created #420

**Comment by duckontheweb** [comment]
Closed by #420
